cordova-plugin-outline/apple/vpn/Shadowsocks.m

```diff
@@ -61,6 +61,7 @@ @interface Shadowsocks () <GCDAsyncSocketDelegate, GCDAsyncUdpSocketDelegate>
 @property (nonatomic) bool serverCredentialsAreValid;
 @property (nonatomic) bool isServerReachable;
 @property (nonatomic) int udpForwardingNumChecks;
+@property(nonatomic) bool performConnectivityChecks;
```

Comment:
Nit: space

Reply:
Done. clang-format should have caught it :/